### PR TITLE
Add new examples

### PR DESCRIPTION
**Pull request by ercius**
Removed some old files in the examples folder.

Add script for electron counting currently used on the NCEM edge compute:
`electron_count.py`: The currently used script to count data sets on the NCEM edge compute. Uses `argparse` instead of `click`.

Add 4 new examples of common operations on sparse data:
1. `center_of_mass_sparse.ipynb`: Notebook showing how to calculate the center of mass and the differential phase contrast
2. `radial_sum_display.ipynb`: Radian sum of a sparse data set with graphical interactive display
3. `sum_scans.ipynb`: Add sparse datasets together all in the sparse domain.
4. `vacuum_scan_subtract.ipynb`: Remove the shift of the beam on the detector with known positions from a vacuum scan

### CODE REVIEW

File: examples/center_of_mass_sparse.ipynb [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+    "    # Calculate numerator and denominator expressions for solution of \n",
+    "    # phase from centre of mass measurements\n",
+    "    numerator = ky[:,None]*np.fft.fft2(CoMx_rot)+kx[None,:]*np.fft.fft2(CoMy_rot)\n",
+    "    denominator = 2*np.pi*1j*((kx**2)[None,:]+(ky**2)[:,None])+reg\n",
+    "    # Avoid a divide by zero for the origin of the Fourier coordinates\n",
+    "    numerator[0,0] = 0\n",
+    "    denominator[0,0] = 1\n",
+    "\n",
+    "    # Return real part of the inverse Fourier transform\n",
+    "    return np.real(np.fft.ifft2(numerator/denominator))\n"
```
Comment (cjh1) [inline examples/center_of_mass_sparse.ipynb:86]: I guess this can be replace with a call the stempy, now that you added it?

Comment (ercius) [reply examples/center_of_mass_sparse.ipynb:86]: yes. That was the plan.  I forgot to mention that. Thanks!

Just please give credit to [py4DSTEM](https://github.com/py4dstem/py4DSTEM/blob/72f95b2c4a5b1d62ee9e491df88a7c3ba7487261/py4DSTEM/process/dpc/dpc.py#L219) and Hamish Brown.

Comment (ercius) [reply examples/center_of_mass_sparse.ipynb:86]: This is added #199

Comment (cjh1) [reply examples/center_of_mass_sparse.ipynb:86]: OK, we can go this in a follow up PR.

**Review by cjh1 — APPROVED** [review]

File: examples/center_of_mass_sparse.ipynb (hunk 1 above)
Comment (ercius) [reply examples/center_of_mass_sparse.ipynb:86]: Once the phase_from_com is in stempy I can test it and remove this.

**Comment by ercius** [comment]
I resolved the conflict in stempy.image. This was placed in the wrong branch because its not part of the "examples cleanup". I do not think it will conflict with the other PRs I made recently.

**Pull request merged**